reframe/core/pipeline.py

```diff
@@ -734,10 +738,16 @@ def __init__(self):
         pass
 
     @classmethod
-    def __init_subclass__(cls, *, special=False, **kwargs):
+    def __init_subclass__(cls, *, special=False, base_test=False, **kwargs):
```

Comment:
I don't like very much `base_test` as a name. It can be confusing when writing a test. I  kind of prefer something like `template`, although it is not ideal, too.

Reply:
I've changed (for now 😂) the `base_test` argument to `pin_prefix`.